LoopKitUI/Views/Settings Editors/TherapySettingsViewModel.swift

```diff
@@ -79,9 +83,10 @@ public class TherapySettingsViewModel: ObservableObject {
         therapySettings.workoutTargetRange = workout?.doubleRange(for: unit)
         didSave?(TherapySetting.workoutCorrectionRangeOverride, therapySettings)
     }
-    
-    public func saveSuspendThreshold(value: GlucoseThreshold) {
-        therapySettings.suspendThreshold = value
+
+    public func saveSuspendThreshold(quantity: HKQuantity) {
+        let settingsGlucoseUnit = therapySettings.glucoseUnit ?? .milligramsPerDeciliter
```

Comment:
[nit] Isn't this the same as the above computed property `therapySettingsGlucoseUnit`?